src/hooks/use-document-caption.tsx

```diff
@@ -17,7 +17,11 @@ export function useDocumentCaption(document: DocumentModelType, isStudentWorkspa
     || (document.isRemote ? teacher?.name : "")
     || "Unknown User";
 
-  const hasNamePrefix =  document.isRemote || isPublishedType(type) || isUnpublishedType(type) || isStudentWorkspaceDoc;
+  const hasNamePrefix =  document.isRemote
+    || isPublishedType(type)
+    || isUnpublishedType(type)
+    || isSortableType(type)
```

Comment:
I don't think it is a good idea to use `isSortableType` here. If we start sorting additional document types it would be unexpected that this would affect whether a name prefix is shown.

Reply:
@scytacki, I don't know if what I changed is the ideal.  There are a lot of factors and conditions that contribute to what each document type shows in its caption.  But I think I understand what you mean now - that "sortable" should be a separate concern from caption.  So I just did ` || type === "exemplar"`. But that is still adding another speical case... so Let me know if I am still missing the mark.